sdk/eventhub/azure-eventhub-checkpointstoretable/azure/eventhub/extensions/checkpointstoretable/_tablestoragecs.py

```diff
@@ -22,17 +27,246 @@ class TableCheckpointStore:
         The hostname of the secondary endpoint.
     """
 
-    def __init__(self, **kwargs):
-        pass
+    def __init__(self, table_account_url, table_name, *, credential=None, **kwargs):
```

Comment:
🤔 why is there an arg `*`? @Jg1255 , I think this is causing an error and should be removed:
```suggestion
    def __init__(self, table_account_url, table_name, credential=None, **kwargs):
```

Reply:
good catch